src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/TilbakekrevingMapper.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.familie.ef.iverksett.tilbakekreving
+
+import no.nav.familie.ef.iverksett.iverksetting.domene.Tilbakekrevingsdetaljer
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.kontrakter.felles.Fagsystem
+import no.nav.familie.kontrakter.felles.Språkkode
+import no.nav.familie.kontrakter.felles.arbeidsfordeling.Enhet
+import no.nav.familie.kontrakter.felles.tilbakekreving.*
+
+class TilbakekrevingMapper {
+
+    fun map(iverksett: Iverksett, enhet: Enhet): OpprettTilbakekrevingRequest {
+        return OpprettTilbakekrevingRequest(
+                fagsystem = Fagsystem.EF,
+                ytelsestype = Ytelsestype.OVERGANGSSTØNAD,
+                eksternFagsakId = iverksett.fagsak.fagsakId.toString(),
+                personIdent = iverksett.søker.personIdent,
+                eksternId = iverksett.behandling.eksternId.toString(),
+                behandlingstype = Behandlingstype.REVURDERING_TILBAKEKREVING,
+                manueltOpprettet = false,
+                språkkode = Språkkode.NB,
+                enhetId = enhet.enhetId,
+                enhetsnavn = enhet.enhetNavn,
+                saksbehandlerIdent = iverksett.vedtak.saksbehandlerId,
+                varsel = lagVarsel(iverksett.vedtak.tilbakekreving ?:
+                                   error("Iverksett.vedtak.feilutbetaling er påkrevd for å map'e Iverksett til OpprettTilbakekrevingRequest")),
+                revurderingsvedtaksdato = iverksett.vedtak.vedtaksdato,
+                verge = null,
+                faktainfo = lagFaktainfo(iverksett)
+        )
+    }
+
+    private fun lagFaktainfo(iverksett: Iverksett): Faktainfo {
+        return Faktainfo(
+                revurderingsårsak = iverksett.behandling.behandlingÅrsak.toString(),
+                revurderingsresultat = iverksett.vedtak.vedtaksresultat.toString(),  // Er dette korrekt?
```

Comment:
Kanskje ryddigere om FaktaInfo bruker enumene behandlingårsak og vedtaksresultat i stedet for String, sålenge det er bare de verdiene som skal brukes her

Reply:
Faktainfo er definert av familie-tilbake, som har satt det til String. De må eventuelt definere enums i protokollen, som vi kan mappe til. Setter foreløpig til String